### ISSUE TITLE
actions.chain do not clean after actions.perform

### ISSUE THREAD

**Issue by FreemanDan** [droefs/HLISA#3]
**TEST CODES**
```
from selenium import webdriver
from HLISA.hlisa_action_chains import HLISA_ActionChains as HLISA


class Webdriver:
    def __init__(self):
        self.driver = Webdriver.create_driver_with_options(self)
        self.actions = HLISA(self.driver)
    def __del__(self):  
        self.driver.quit()
           
    @staticmethod
    def create_driver_with_options(self):
        driver = webdriver.Chrome('chromedriver\chromedriver.exe')
        return driver

mydriver = Webdriver()
mydriver.driver.get('https://google.com')
element = mydriver.driver.find_element('xpath', '//img[@alt="Google"]')
print(element)
mydriver.actions.move_to_element(element).perform()
print('chain:')
print(mydriver.actions.chain)
```
**OUTPUT**
```
chain:
[<function HLISA_ActionChains.move_to_element.<locals>.<lambda> at 0x0000022203A37430>, <function HLISA_ActionChains.move_to_element.<locals>.<lambda> at 0x00000222125D4820>]
```

**Comment by droefs** [comment]
Thank you for pointing out this bug and providing a clear working example!

The issue is resolved in HLISA 1.3.1 on PyPI.

Please note that if you are using a selenium version before 4, the behavior is unchanged, as this is how .perform() worked in selenium versions prior to 4. You can use `reset_actions()` after `perform()` in that case. If you are using selenium 4 however, it should now work as you would expect.

**Comment by FreemanDan** [comment]
Thank you for fixing selenium 4 compatibility so soon!
